fontbe/src/os2.rs

```diff
@@ -10,6 +12,187 @@ use crate::{
     orchestration::{BeWork, Context},
 };
 
+/// Used to build a bitfield.
+///
+/// Intended use: use the outer range to determine if a given bit might apply. If it might,
+/// check the inner ranges.
```

Comment:
the comment is no longer in sync with the implementation below I think, I don't see outer/inner ranges anymore, just a single array of 3-tuples